.github/workflows/ci.yml

```
@@ -34,3 +34,12 @@ jobs:
 
       - name: Lint rust
         run: cargo clippy && cargo fmt --check
+
+      - name: Smoke test gem install
+        shell: bash
+        run: |
+          bundle exec rake build
+          gem install pkg/wasmtime-rb-*.gem --verbose
+          script="puts Wasmtime::Engine.new(Wasmtime::Config.new).precompile_module('(module)')"
```

Nit: the config is now, this should also work:
```suggestion
          script="puts Wasmtime::Engine.new.precompile_module('(module)')"
```